src/pijama_ast/src/location/mod.rs

```diff
@@ -1,8 +1,12 @@
+//! Utilities for capturing and representing the location
+//! of tokens in the source code file.
 use std::fmt::{Debug, Display, Formatter, Result as FmtResult};
 
+/// Type representing a length of tokens.
```

Comment:
`Span` also represents the location of a token. You can get where the token starts and how long it is.